### DISCUSSION TITLE
Welcome to GBE Discussions!

### DISCUSSION THREAD

**Discussion by bethlakshmi** [bethlakshmi/gbe-divio-djangocms-python2.7#488] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by burlexpo** [comment]
Hunh.  Look at that! Discussions!

Hello everyone, I'm Scratch.  I'm the founding chair and producer of the event.  The buck stops here... which now that that I'm thinking about it is a kind of weird expression.
